recipes/Hydrostats/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "hydrostats" %}
+{% set version = "0.75" %}
+{% set sha256 = "64910ad69e5cfcaa96e8b7f5c8da34666fcd76bb9f5d8546ca0b3e6898fc3614" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - numpy
+    - scipy
+    - numba
+    - pandas
+    - matplotlib
+    - HydroErr
```

Package names use lowercase letters:

```suggestion
    - hydroerr
```